tests/jupyterhub_api/test_check_api.py

```diff
@@ -0,0 +1,79 @@
+"""
+Test if a api is reachable
+"""
+
+import json
+import requests
+import subprocess
+import os
+import docker
+import sys
+sys.path.append("..")
+import time
+from logger import Logger, LOG_FOLDER, LOG_EXTENSION
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
+requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
+
+import argparse
+
+def get_args():
+    parser = argparse.ArgumentParser(description='Arguments', formatter_class = argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument("--port", dest="port", type=int,
+                        required = True,
+                        )
+    args = parser.parse_args()
+    return args
+
+
+class CheckAPI:
```

Comment:
Like the storage, inherit this from some Test class that implements all this repeated code (logic for the logs and the log_params function all together).